packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/cloudwatch_logger_plugin.dart

```diff
@@ -295,3 +364,34 @@ extension on LogEntry {
     );
   }
 }
+
+extension on RejectedLogEventsInfo {
+  (int? pastEndIndex, int? futureStartIndex) parse(int length) {
+    int? pastEndIndex;
+    int? futureStartIndex;
+
+    if (_isValidIndex(tooOldLogEventEndIndex, length)) {
```

Comment:
Why is this allowed to be false? I would use `RangeError.checkValidIndex` to assert these are valid.